store/CSDK/CarbonReader.cpp

```diff
@@ -38,9 +37,9 @@ jobject CarbonReader::builder(JNIEnv *env, char *path) {
     jclass carbonReaderClass = env->FindClass("org/apache/carbondata/sdk/file/CarbonReader");
```

Comment:
same as Above, for all the class in carbonReader.cpp handle the negative scenarios by adding validation. Else we get jvm crash.

Reply:
added